File: leetcode/191-number-of-1-bits.cpp
class Solution {
public:
    int hammingWeight(int n) {
        int w = 0;

        for(int i =0; i<32;i++)
        {
            if( ((n>>i) % 2) == 1)
                w++;
            
        }
        return w;
    }
};

File: leetcode/191-number-of-1-bits.py
class Solution(object):
    def hammingWeight(self, n):
        """
        :type n: int
        :rtype: int
        """
        w = 0
        for i in range(0,31):
            if (n>>i)%2 == 1:
                w = w+1
        
        return w
        